examples/base_time_add/time_solver.py

```diff
@@ -0,0 +1,75 @@
+from time import sleep
+
+from sidecar_solver import command_add
+from sidecar_solver import command_get_time
+from sidecar_solver import command_print_solver_status
+from sidecar_solver import control_interface
+
+from osparc_control.core import ControlInterface
+from osparc_control.models import CommandRequest
+
+
+def handle_inputs(time_solver: "TimeSolver", request: CommandRequest) -> None:
+    if request.action == command_add.action:
+        sum_result = time_solver._add(**request.params)
+        time_solver.control_interface.reply_to_command(
+            request_id=request.request_id, payload=sum_result
+        )
+        return
+
+    if request.action == command_get_time.action:
+        time_solver.control_interface.reply_to_command(
+            request_id=request.request_id, payload=time_solver.time
+        )
+        return
+
+    if request.action == command_print_solver_status.action:
+        print("Solver internal status", time_solver)
+        # finally exit
+        time_solver.can_continue = False
```

Comment:
print solver exits the solver?
Would it not be more clear to have an exit command instead?

Reply:
Will change message, but will not use exit. This gives time to close background threads.